ceph.spec.in

```diff
@@ -1945,5 +1957,9 @@ exit 0
 # actually build this meta package.
 %endif
 
+%files grafana-dashboards
+%attr(0755,root,root) %dir %{_sysconfdir}/grafana/dashboards/ceph-dashboard
+%config %{_sysconfdir}/grafana/dashboards/ceph-dashboard/*
```

Comment:
might want package `monitoring/grafana/dashboards`. it's difficult to figure out the dependency or how to use these settings without further pointers and instructions.